playwright/user-stories/view-items.md

```diff
@@ -8,3 +8,71 @@
 Given there are multiple volumes in an item
 When the researcher views the item
 Then the volumes should be browsable
+
+Title: A user I can view an item page
+
+As a researcher
+I need to evaluate things I am interested in
+so that I can determine whether they are relevant to my research
+
+Scenario 1: A user is viewing a digitised item (with a iiif-presentation location)
+
+Given a user is viewing a digitised book
+Then that page will be a valid Item page
+And that page will include:
+The title of the work
+The contributors of the work
+The date of the work
+License and credit information
+Full screen button
```

Comment:
This is becoming very implementation dependant, and not focusing on the user needs behind these features.

You might have a user need of being able to view an item at a larger scale, so we test the zoom and fullscreen buttons.

Similarly for downloads, the user need is to have content offline to be able to use elsewhere, so we allow them to download it.